dojo/product_type/views.py

```diff
@@ -48,13 +48,22 @@ def prefetch_for_product_type(prod_types):
 
     if isinstance(prefetch_prod_types, QuerySet):  # old code can arrive here with prods being a list because the query was already executed
         active_findings_query = Q(prod_type__engagement__test__finding__active=True,
+                                prod_type__engagement__test__finding__mitigated__isnull=True,
+                                prod_type__engagement__test__finding__verified=False,
+                                prod_type__engagement__test__finding__false_p=False,
+                                prod_type__engagement__test__finding__duplicate=False,
+                                prod_type__engagement__test__finding__out_of_scope=False)
```

Comment:
all these should go, we should stick to the definitions made earlier and used in https://github.com/DefectDojo/django-DefectDojo/blob/c67c3feb86fe778aac115fcfb93034f1721add2e/dojo/finding/views.py